includes/class-wc-subscriptions-order.php

```diff
@@ -1323,6 +1331,35 @@ public static function map_order_query_args_for_subscriptions( $query_vars ) {
 		return $query_vars;
 	}
 
+	/**
+	 * Modifies the query clauses of a wc_get_orders() query to include/exclude parent orders based on the 'subscription_parent' argument.
+	 *
+	 * @param array            $query_clauses The query clauses.
+	 * @param OrdersTableQuery $order_query   The order query object.
+	 *
+	 * @return $query_clauses The modified query clauses to include/exclude parent orders.
+	 */
+	public static function filter_orders_query_by_parent_orders( $query_clauses, $order_query ) {
+		$include_parent_orders = $order_query->get( 'subscription_parent' );
+
+		// Bail if there's no argument to include/exclude parent orders.
+		if ( is_null( $include_parent_orders ) ) {
+			return $query_clauses;
+		}
+
+		if ( true === $include_parent_orders ) {
+			// Limit query to parent orders.
+			$query_clauses['join']  = ( empty( $query_clauses['join'] ) ? '' : $query_clauses['join'] . ' ' );
+			$query_clauses['join'] .= "INNER JOIN {$order_query->get_table_name( 'orders' )} as subscriptions ON subscriptions.parent_order_id = wp_wc_orders.id AND subscriptions.type = 'shop_subscription'";
```

Comment:
```suggestion
			$query_clauses['join'] .= "INNER JOIN {$order_query->get_table_name( 'orders' )} as subscriptions ON subscriptions.parent_order_id = {$order_query->get_table_name( 'orders' )}.id AND subscriptions.type = 'shop_subscription'";
```

I noticed the orders table was hardcoded here. I've been testing with this suggestion and it works

Reply:
Oh good catch. Thanks for the suggestions.